client/src/store/settings.ts

```diff
@@ -35,11 +35,31 @@ const showPopover = atom<boolean>({
   default: false,
 });
 
+const autoScroll = atom({
+  key: 'autoScroll',
+  default: localStorage.getItem('autoScroll') === 'true',
+  effects_UNSTABLE: [
```

Comment:
based on this discussion, we can just use `effects` here unless it doesn't work for you: https://github.com/facebookexperimental/Recoil/discussions/1925